### DISCUSSION TITLE
Copying an Iris cube list?

### DISCUSSION THREAD

**Discussion by P-Burns** [SciTools/iris#5140] [Q&A] [answered]
Hi,

Is there a safe method to copy an Iris cube list, so that when I modify the new cube list it doesn't change the contents of the copied cube list?  It appears that the copy() method is only for Iris cubes and not cube lists.  

Many thanks,
Paul

**Comment by trexfeathers** [comment, marked as answer]
This was discussed in #3651, and it was decided that `CubeList` should behave as close to `list` as possible. As such, if you want the members of the new `CubeList` to be truly different from the originals, you need to use `deepcopy()`. [See this Stack Overflow answer](https://stackoverflow.com/a/2612815).

```python
from copy import deepcopy

new_cube_list = deepcopy(my_cube_list)

```

**Comment by P-Burns** [comment]
That's great.  Many thanks for your clear response.
